packages/install-expo-modules/src/plugins/android/__tests__/withAndroidModulesMainActivity-test.ts

```diff
@@ -100,7 +100,165 @@ class MainActivity : ReactActivity() {
     expect(nextContents).toEqual(expectContents);
   });
 
-  it(`should add ReactActivityDelegateWrapper - java`, () => {
+  it(`should add ReactActivityDelegateWrapper for react-native@>=0.68.0 - java`, () => {
+    const rawContents = `
+package com.helloworld;
```

Comment:
xit: I would prefer if this larger fixtures were in external files so there's no risk of the format breaking.